### DISCUSSION TITLE
I would like to suggest new segment functions

### DISCUSSION THREAD

**Discussion by Bindkedon** [mifi/lossless-cut#2246] [Ideas]
The function I would like to add is "select this segment" (without deselecting all others). Ideally it would be executable with a keyboard shortcut.
The reason:
You split a video with sequence recognition and 300 segments are created. You want to keep 10 - 20 and delete the rest.
So you go through the timeline and pick the ones you want to keep. The only option so far is to go to the right column each time and deselect the desired segment. At the end you can invert the entire column and then have the segments you actually want. It would be much easier and quicker to deselect all the segments directly, then go through the timeline and select the ones you want, done.
The  commands of now do not allow for such a way of working.
Many thanks...

**Comment by Bindkedon** [comment]
sorry I meant scene recognition...

**Comment by mifi** [comment]
Hi. Isn't what you're looking for "Toggle current segment selected"/toggleCurrentSegmentSelected which already exists?

**Reply by Bindkedon** [reply]
Sorry 'Toggle current segment selected' I cannot find... You're right that would solve the problem... but where is it?

**Reply by mifi** [reply]
It's in the keyboard shortcuts dialog, and it's the action when clicking the checkbox in the lower right of each segment in the right side segments list

**Reply by Bindkedon** [reply]
Yes, and that is exactly what complicates the workflow. Because you have to go to the side bar every time to find the corresponding segment and click off. It would be much easier and faster to be able to do this directly from the timeline, with a shortcut and/or right click /activate. Or have I missed something? Thanks!

**Reply by mifi** [reply]
But you asked for a keyboard shortcut initially:

> Ideally it would be executable with a keyboard shortcut.

and there is already such a keyboard shortcut, or am i misunderstanding something?

**Reply by Bindkedon** [reply]
Okay, I did'nt find it. Please can you tell me which it is? Shortcut to activate or disactivate the segment where the cursor in the timeline is just about.

**Reply by mifi** [reply]
It's called "Toggle current segment selected" in the Keyboard shortcuts list. you can search for it. Or are you looking for a new feature: "Toggle selected all segments which intersect with the timeline cursor"?

**Reply by Bindkedon** [reply]
Actually it is there! I apologize for disturbing you, I really haven't found it yet. Many Thanks! I don't necessarily need the other option.
But I have one more question if I may... :
So far I have always worked with Avidemux in copy mode. Avidemux cuts 100% cleanly. No artifacts were ever created in any further processing programs. How should losslesscut be set so that it also cuts reliably at keyframes? There are simply a lot of different setting options here, which ones should be set how? I've already read through all the FAQs but haven't really figured it out yet. Thank you very much!

**Reply by mifi** [reply]
that's very hard for me to say also. i haven't found a solution to that problem. more discussion #1216
